### PR TITLE
add HyperVGeneation in GetVMImage

### PR DESCRIPTION
**Pull request by khan15**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -6046,6 +6058,9 @@
         },
         "automaticOSUpgradeProperties": {
           "$ref": "#/definitions/AutomaticOSUpgradeProperties"
+        },
+        "hyperVGeneration": {
+          "$ref": "#/definitions/HyperVGeneration"
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Why don't you use the existing HyperVGenerationType, if enum values are the same?

**Comment by NullMDR** [comment]
@openapibot sdkautomation rebuild

**Comment by hyonholee** [comment]
LGTM

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -6046,6 +6046,9 @@
         },
         "automaticOSUpgradeProperties": {
           "$ref": "#/definitions/AutomaticOSUpgradeProperties"
+        },
+        "hyperVGeneration": {
```
Comment (majastrz) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: >hyperVGeneration [](start = 9, length = 16)

Can you confirm that this is a read-only property?

**Review by majastrz — COMMENTED** [review]
Can you confirm that the new property is read-only?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 2 above)
Comment (erich-wang) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @khan15, could you confirm?

**Comment by hyonholee** [comment]
LGTM

**Comment by majastrz** [comment]
> This property only appears in response body. So we don't need to set it as read-only, right?

Yeah, that's fine. I was asking the question to confirm that this is not a breaking change. Based on your answer (it only shows up on GET responses), it's not a breaking change.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Pull request merged**